content/e/ddw/2015-03-14-ddw-house-show-1.md

```diff
@@ -21,9 +32,39 @@ venue = ["gimnazjum-8-gdansk"]
   - '[Piękny Kawaler](@/w/piekny-kawaler.md)'
   - s: "Three Way Match"
 - credits:
+    Ring announcer: '[Arkadiusz Pawłowski](@/w/pan-pawlowski.md)'
     Referee: '[Krystian Malinowski](@/w/krystian-malinowski.md)'
 {% end %}
 
+#### Battle Royal
+
+{% inline_fig(path="2015-03-14-ddw-house-show-1-battle-royal-1.jpg") %}
+Left to right: \
+_rookie_ [Sawicki](@/w/sawicki.md) (in the left corner) \
+_rookie_ [Rex Torpeda](@/w/krystian-malinowski.md) (kicking _rookie_ Sawicki) \
+[Greg](@/w/greg.md) (behind _rookie_ Rex Torpeda, in white T-shirt) \
+_rookie_ [Damian Lambert](@/w/damien-rothschild.md) (being thrown out by Greg) \
```

Comment:
this one doesn't need to be `_italic_` - it's just text. Only in the matchcard this is important.